tests/src/rules/no-duplicates.js

```diff
@@ -402,7 +401,7 @@ import {x,y} from './foo'
     // #2027 long import list generate empty lines
     test({
       code: "import { Foo } from './foo';\nimport { Bar } from './foo';\nexport const value = {}",
-      output: "import { Foo , Bar } from './foo';\nexport const value = {}",
+      output: "import { Bar, Foo } from './foo';\nexport const value = {}",
```

Comment:
this suggests it's a breaking change. Why is this order reversed? Foo appears first in the original source code; it should appear first in the autofix.